sys/include/ffs/fs.h

```diff
@@ -511,9 +511,7 @@ typedef struct Cg {
 #define	cgimin(fs, c)	(cgstart(fs, c) + (fs)->fs_iblkno)	/* inode blk */
 #define	cgsblock(fs, c)	(cgstart(fs, c) + (fs)->fs_sblkno)	/* super blk */
 #define	cgtod(fs, c)	(cgstart(fs, c) + (fs)->fs_cblkno)	/* cg block */
-#define	cgstart(fs, c)							\
-       ((fs)->fs_magic == FS_UFS2_MAGIC ? cgbase(fs, c) :		\
-       (cgbase(fs, c) + (fs)->fs_old_cgoffset * ((c) & ~((fs)->fs_old_cgmask))))
+#define	cgstart(fs, c)	(cgbase(fs, c))
```

Comment:
this is not UFS1_MAGIC, it's UFS2

Reply:
For our file system, fs_magic will be FS_UFS2_MAGIC, so we can just use the first expression (cgbase(fs, c))